### ISSUE TITLE
'horizon' in forecast_cases()

### ISSUE THREAD

**Issue by maressyl** [epiforecasts/EpiSoon#42]
Hi,

As we are working on developing unit tests for the package functions (and considering I'm quite new to epidemiology), I could use some clarification regarding the behavior expected from forecast_cases().

I start here from the examples in forecast_cases() manual :

```
forecast <- forecast_rt(
	EpiSoon::example_obs_rts[1:10,],
	model = function(...) {
		EpiSoon::bsts_model(
			model = function(ss, y){
				bsts::AddAutoAr(ss, y=y, lags=10)
			},
			...
		)
	},
	horizon = 7,
	samples = 10
)

set.seed(42)
forecast_cases(
	EpiSoon::example_obs_cases,
	fit_samples = forecast,
	serial_interval = EpiSoon::example_serial_interval,
	horizon = 1
)

set.seed(42)
forecast_cases(
	EpiSoon::example_obs_cases,
	fit_samples = forecast,
	serial_interval = EpiSoon::example_serial_interval,
	horizon = 5
)

set.seed(42)
forecast_cases(
	EpiSoon::example_obs_cases,
	fit_samples = forecast,
	serial_interval = EpiSoon::example_serial_interval,
	horizon = 10
)
```

Isn't the output expected to be limited to rows with relevant "horizon" values ? How should forecast_cases() behave when horizon exceed the value provided to fit `forecast` ?

I am working with a version pulled from Github last week (8eddc07189c08d5759c50b9de11c946f7049537a).

Best regards,
Sylvain

**Comment by seabbs** [comment]
Hi Sylvain, 

My understanding of this function is that horizon is the maximum to explore (so everything between 1:horizon should be returned) That being said in nearly all use cases the behaviour that is used is to produce a case forecast over the same time horizon as the input Rt forecast. 

(Thanks for all the work on the package so far btw and sorry for the slow response). 

Sam

**Comment by sbfnk** [comment]
Fixed in #76
